src/schema/objects/metadata.yaml

```diff
@@ -2554,6 +2689,18 @@ RecordingType:
     - continuous
     - epoched
     - discontinuous
+RecordTimeStart:
+  name: RecordTimeStart
+  display_name: Eye-tracking recording start time
+  description: |
+    Eye-tracking timestamp corresponding to the onset (start) of the run.
+  type: number
+RecordTimeEnd:
+  name: RecordTimeEnd
+  display_name: Eye-tracking recording start time
+  description: |
+    Eye-tracking timestamp corresponding to the offset (end) of the run.
+  type: number
```

Comment:
related backref: good old issue/discussion about establishing the synchronization of the clocks/acquisitions across different modalities: https://github.com/bids-standard/bids-specification/issues/86

Reply:
I will read this and discuss it with @Remi-Gau 
Thanks